src/main/resources/db/liq/changelog.xml

```diff
@@ -0,0 +1,163 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+        https://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.8.xsd">
+    <!-->columns ? username <-->
+    <changeSet id="1" author="aleksdenni">
+        <createTable tableName="users">
```

Comment:
users -> change to user   (all table names & fileds should be as in the  structure (created by mentor)